### PR TITLE
readme: remove stale ROADMAP and add Community section, etc.

### PR DESCRIPTION
**Pull request by cofyc**
### What problem does this PR solve? 
- remove stable ROADMAP.md
- add 'Community' section
- update links
### Check List 

Tests 

 - Unit test
 - E2E test
 - Stability test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,20 +45,29 @@ TiDB Operator manages [TiDB](https://github.com/pingcap/tidb) clusters on [Kuber
...
+- [简体中文](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/tidb-operator-overview)
+
+All the TiDB Operator documentation is maintained in the [docs-tidb-operator repository](https://github.com/pingcap/docs-tidb-operator). 
+
+## Community
+
+Feel free to reach out if you have any questions. The maintainers of this project are reachable via:
+
+- [TiDB Community Slack](https://pingcap.com/tidbslack/) in the [#sig-k8s](https://tidbcommunity.slack.com/archives/CHD0HA3LZ) channel
+- [Filling an issue](https://github.com/pingcap/tidb-operator/issue) against this repo
```
Comment (DanielZhangQD) [inline README.md:66]: Is `File an issue` better?

File: README.md [hunk 2]
```diff
@@ -45,20 +45,29 @@ TiDB Operator manages [TiDB](https://github.com/pingcap/tidb) clusters on [Kuber
...
 
-## Quick start
+## Documentation
 
-Read the [Quick Start Guide](https://pingcap.com/docs/v3.0/tidb-in-kubernetes/tidb-operator-overview/), which includes all the guides for managing TiDB clusters in Kubernetes.
+You can see our documentation at PingCAP website for more in-depth installation and instructions for production:
 
-## Documentation
+- [English](https://docs.pingcap.com/tidb-in-kubernetes/stable/tidb-operator-overview)
+- [简体中文](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/tidb-operator-overview)
```
Comment (ran-huang) [inline README.md:57]:
```suggestion
- [English](https://docs.pingcap.com/tidb-in-kubernetes/stable)
- [简体中文](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable)
```
How about using the landing page?

File: README.md (hunk 1 above)
Comment (cofyc) [reply README.md:66]: "filing an issue" should be better
(if they are concatenated, `are reachable via filing an issue`)

File: README.md (hunk 2 above)
Comment (cofyc) [reply README.md:57]: SG

File: README.md (hunk 1 above)
Comment (DanielZhangQD) [reply README.md:66]: OK, then please fix the typo `Filing`.

Comment (cofyc) [reply README.md]: oh, thanks! fixed

**Comment by cofyc** [comment]
@ran-huang fixed a typo, PTAL

**Pull request merged**